pkg/actions/apply/apply.go

```diff
@@ -0,0 +1,63 @@
+package apply
+
+import (
+	"fmt"
+
+	"github.com/kris-nova/logger"
```

Comment:
Any regard for ordering imports? 

I like the structure:
- stdlib
- third party
- under orgranization
- in the same repo

What do you think?

Reply:
great question! This is not something I've really ever "cared" too much for, I'd be happy for us to start taking an opinion. I wish the built in linter had the opinion for us :disappointed:

Reply:
Yeah no linter check other then goimport -local I think. :/ But the IDE let's you set up import grouping format which is nice.